src/main/java/org/minions/devfund/miguelcalderon/WhackAMole.java

```diff
@@ -0,0 +1,216 @@
+package org.minions.devfund.miguelcalderon;
+
+import java.util.Random;
+import java.util.Scanner;
+
+/**
+ * Solution for "Whackmole game" excercise.
+ *
+ * @author Miguel Calderon
+ * @version 1.0 16 Jul 2018
+ */
+
+public class WhackAMole {
+    private int molesLeft;
+    private int attemptsLeft;
+    private char[][] moleGrid;
+    private int score;
+    private static final char MOLE = 'M';
+    private static final char WHACKED_MOLE = 'W';
+    private static final char EMPTY = '*';
+
+    /**
+     * Constructor.
+     *
+     * @param numAttempts   number of attempts.
+     * @param gridDimension number of rows and columns of the grid.
+     */
+    public WhackAMole(int numAttempts, int gridDimension) {
+        this.moleGrid = new char[gridDimension][gridDimension];
+        attemptsLeft = numAttempts;
+        this.molesLeft = 0;
+        score = 0;
+        createEmptyGrid();
+        setRandomPlaceForMoles();
+
+    }
+
+    /**
+     * Creates empty grid.
+     */
+    public void createEmptyGrid() {
+        for (int i = 0; i < moleGrid.length; i++) {
+            for (int j = 0; j < moleGrid.length; j++) {
+                moleGrid[i][j] = EMPTY;
+            }
+        }
+    }
+
+    /**
+     * Set moles in a cell.
+     *
+     * @param x axis.
+     * @param y axis.
+     * @return True if the Mole was added properly otherwise false.
+     */
+
+    public boolean place(int x, int y) {
+        //if (x >= 0 && y >= 0 && x < moleGrid.length && y < moleGrid.length)
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)

Reply:
block deleted